pkg/apiserver/httprequest/httprequest.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"strings"
 
-	"bitbucket.org/ww/goautoneg"
```

Comment:
I would expect a replace in go.mod.

What is upstream kube doing?

Reply:
replace doesn't work, upstream uses github.com/munnerz/goautoneg

```
go: github.com/munnerz/goautoneg@v0.0.0-20191010083416-a7dc8b61c822 used for two different module paths (bitbucket.org/ww/goautoneg and github.com/munnerz/goautoneg)
```

Reply:
https://github.com/kubernetes/kubernetes/blob/2d8fbd61afe9e2bf207be88320fd9c3001885a96/go.mod#L75